python/lsst/ts/linearstage/csc.py

```diff
@@ -191,6 +202,8 @@ def assert_target_in_range(self, target_value, move_type, axis):
         salobj.ExpectedError
             Raised when the command is not allowed in the current state.
         """
+        if self.component is None:
+            raise RuntimeError("Component is not setup.")

```

Comment:
It is shorter to use `assert self.component is not None` which is the accepted design pattern for TSSW projects.